client/session.go

```diff
@@ -361,6 +362,8 @@ func (s *session) setTopologyWithLock(topologyMap topology.Map, queues []hostQue
 			prev[i].Close()
 		}
 	}()
+
+	s.log.Infof("successully updated topology to %v hosts", topologyMap.HostsLen())
```

Comment:
`successully` -> `successfully`, also while you are there, might change `%v` to `%d`?